src/TBuffer.cpp

```diff
@@ -4136,7 +4164,7 @@ bool TBuffer::processGBSequence(const std::string& bufferData, const bool isFrom
                 if        (  (static_cast<quint8>(bufferData.at(pos    )) >= 0x81) && (static_cast<quint8>(bufferData.at(pos    )) <= 0xA0)
                           && (static_cast<quint8>(bufferData.at(pos + 1)) >= 0x40) && (static_cast<quint8>(bufferData.at(pos + 1)) <= 0xFE)
                           && (static_cast<quint8>(bufferData.at(pos + 1)) != 0x7F) ) {
-                    //clang-format on
+                    // clang-format on
```

Comment:
I've switched off the clang-formatting  so that all the `if` tests are laid out consistently and in a way that makes it a bit easier for humans to parse - it is just that some did not have the same spacing...